webauth/index.js

```diff
@@ -52,7 +52,7 @@ export default class WebAuth {
    *
    * @memberof WebAuth
    */
-  authorize(options = {}) {
+  authorize({ disableAuthenticatedSession, ...options } = {}) {
```

Comment:
Why do you need to define it here? Assume is present inside the `options` object and that's it. This is only used on iOS and this method is for a generic webauth.